2_process/src/normalize_streamdata.R

```diff
@@ -16,8 +16,9 @@ normalize_streamdata <- function(ind_file, raw_ind_file, sites_ind_file){
     mutate(stage_normalized = (stage - min(stage, na.rm = T)) / (max(stage, na.rm = T) - min(stage, na.rm = T))) %>%
     mutate(flood_stage_normalized = (as.numeric(flood_stage) - min(stage, na.rm = T)) / (max(stage, na.rm = T) - min(stage, na.rm = T))) %>%
     ungroup()
+  norm_stream_sf <- sf::st_as_sf(norm_stream)
```

Comment:
Oh oops, good to know! I thought it the column `geometry` was still an `sf` object so it would be fine. I had been running it like this - weird that it didn't throw any errors at me.

Reply:
Yeah, it occurred to me later last night that my errors might have been because my sf or dplyr packages might be out of date relative to yours...but this seems safer all around anyway